logstash-core/lib/logstash/pipeline.rb

```diff
@@ -164,10 +164,13 @@ def start
     collect_stats
     collect_dlq_stats
 
-    @logger.info("Starting pipeline", default_logging_keys(
-      "pipeline.workers" => settings.get("pipeline.workers"),
-      "pipeline.batch.size" => settings.get("pipeline.batch.size"),
-      "pipeline.batch.delay" => settings.get("pipeline.batch.delay")))
+    pipeline_log_params = default_logging_keys(
+        "pipeline.workers" => settings.get("pipeline.workers"),
+        "pipeline.batch.size" => settings.get("pipeline.batch.size"),
+        "pipeline.batch.delay" => settings.get("pipeline.batch.delay"))
+
+    pipeline_log_params["pipeline.sources"] = pipeline_source_details
```

Comment:
Can you also add this to https://github.com/elastic/logstash/blob/55783109493adf1bdc425ad4695207fce60bc447/logstash-core/lib/logstash/pipeline.rb#L186